kubernetes_asyncio/leaderelection/example.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+import os
+import uuid
+
+from kubernetes_asyncio import config
+from kubernetes_asyncio.client import api_client
+from kubernetes_asyncio.leaderelection import electionconfig, leaderelection
+from kubernetes_asyncio.leaderelection.resourcelock.configmaplock import (
+    ConfigMapLock,
+)
+
+
+async def main():
```

Comment:
Should this example (and the test file too) be contained within the leaderelection directory, or `examples/` and `test/`?

Reply:
I moved the example, but for the test I noticed that the other tests in `test/` are autogenerated, and that other non-autogenerated tests live next to their code within the repo. I kept the test in-place, but let me know if you'd like it moved (and if so, should I create another PR to move the others?).

Reply:
Sorry, you are correct - all tests (not generated by the generator) are with code files in the same directories, so suffix _test is enough. Thanks.